bridge/cmd/root.go

```diff
@@ -50,19 +54,10 @@ func InitTendermintViperConfig(cmd *cobra.Command) {
 	helper.InitHeimdallConfig("")
 }
 
-// Execute adds all child commands to the root command and sets flags appropriately.
-// This is called by main.main(). It only needs to happen once to the rootCmd.
-func Execute() {
-	if err := rootCmd.Execute(); err != nil {
-		fmt.Println(err)
-		os.Exit(1)
-	}
-}
-
 func init() {
 	var logger = helper.Logger.With("module", "bridge/cmd/")
-	rootCmd.PersistentFlags().StringP(helper.NodeFlag, "n", "tcp://localhost:26657", "Node to connect to")
-	rootCmd.PersistentFlags().String(helper.HomeFlag, os.ExpandEnv("$HOME/.heimdalld"), "directory for config and data")
+	rootCmd.PersistentFlags().StringP(helper.NodeFlag, "n", helper.DefaultNode, "Node to connect to")
```

Comment:
Could we have a more descriptive name of what is this? Is this the connection to the Heimdall daemon? @temaniarpit27 @0xKrishna

Reply:
26657 is the tendermint port

Reply:
`Tendermint` looks too specific, what about `HeimdallNode`?

Reply:
I could change then to ```HeimdallNodeFlag```,  ```DefaultHeimdallNode``` . There will be changes in 7 files because of renaming. Is it ok to keep in server/root.go
```cmd.Flags().String(client.FlagNode, helper.DefaultHeimdallNode, "Address of the node to connect to")```
instead of 
```cmd.Flags().String(helper.HeimdallNodeFlag, helper.DefaultHeimdallNode, "Address of the node to connect to")``` because all other flags names are retrieved from tinderment client package?

Reply:
I have changed name of the flag and default value const to Tendermint and pushed.
I'll mark this as resolved
If I am correct with wrong default value for FlagListenAddr flag we can change it through this or some other task